src/DispatchListener.php

```diff
@@ -109,15 +112,22 @@ public function onDispatch(MvcEvent $e)
 
         $request  = $e->getRequest();
         $response = $application->getResponse();
+        $caughtException = null;
 
         try {
             $return = $controller->dispatch($request, $response);
+        } catch (\Throwable $ex) {
```

Comment:
Shouldn't `Exception` be caught prior to `Throwable`? As Exception extends Throwable and would therefore never be caught?